includes/admin/post-types.php

```diff
@@ -221,6 +225,10 @@ public function edit_table_hooks() {
 	 * @param int $post_id
 	 */
 	private function duplicate_feed( $post_id ) {
+		
+		if(!current_user_can( 'edit_posts' )){
```

Comment:
Same here, do we need to check for `nonce` here?

Reply:
NO, this function call on page reload and take request from url. so there is no need.